### ISSUE TITLE
Final comments to repository

### ISSUE THREAD

**Issue by AliciaMstt** [cristoichkov/GBS_Bioinf_Process_Mamm#27]
What a nice final report and discussion of your data. What I liked the most is that you merged different approaches to GBS assembly optimization, and even proposed new metrics that previous studies haven't looked at. I enjoyed reading it, and the figures are really nice too. Your README is also very clear, the pipeline explanation you added fixed the concerns I had before. Congrats!

Few comments to further improve your README:

* Your English has improved, but you still need to watch out your grammar and correct use of English terms, for instance the correct English term is missing data, not lost data. 
* It is not clear in your discussion what parameters you are finally going to use.
* Keep your github project updated, there are some issues that are already resolved but still appear as "in progress" in your project.

I hope you enjoyed the class. See you in your candidatura ;)

**Comment by cristoichkov** [comment]
Hi, I appreciate your comments!!

I'm taking English classes to improve my grammar and other things.

For the part of choosing the parameters, in the discussion until the end, I wrote this "Therefore, the optimal clustering threshold was set at 0.91 mainly because it is more constant between the metrics and the mindepth at 10. "  The most complicated parameter to choose was "clustering threshold", because in some metrics the optimal values were different.  I'd like to know your opinion about this. 

I'll follow working on my repo. 

Thanks for your support and pieces of advice.

**Comment by AliciaMstt** [comment]
Yeah, I saw that part of your discussion. I agree setting the clustering threshold to .91 is a good choice, and I agree with your reasons. The higher number of loci with .86-.88 vs .91 is not very large, really, so you are not loosing much.

What I mean with "It is not clear in your discussion what parameters you are finally going to use", is that each parameter discussed somewhat independently and the sentece "Therefore, the optimal clustering threshold was set at 0.91 mainly because it is more constant between the metrics and the mindepth at 10. " is in the middle, making it easier for it to get lost. 

I would be useful to 1) break your discussion parragraph in several parragraphs and 2) either start or finish one of these parragraph with the sentence stating which were the optimal parameters you chose then/before you can discuss the reasons why.
